src/Windows/Avalonia.Win32.Automation/AutomationNode.cs

```diff
@@ -355,5 +361,11 @@ private static UiaControlTypeId ToUiaControlType(AutomationControlType role)
                 _ => UiaControlTypeId.Custom,
             };
         }
+
+        private static int GetProcessId()
+        {
+            using var proccess = Process.GetCurrentProcess();
+            return proccess.Id;
+        }
```

Comment:
Consider
```suggestion
        {
#if NET6_0_OR_GREATER
            return Environment.ProcessId;
#else
            using var proccess = Process.GetCurrentProcess();
            return proccess.Id;
#endif
        }
```

Reply:
And then it can be cached in static field as suggested above:
```
private static int s_pid = GetProcessId();
```

Reply:
Thank you. PR is Updated